rules/linux/privilege_escalation_apt_binary.toml

```diff
@@ -0,0 +1,69 @@
+[metadata]
+creation_date = "2022/02/24"
+maturity = "production"
+updated_date = "2022/02/24"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies Linux binary apt/apt-get abuse to breakout out of restricted shells or environments by spawning an
+interactive system shell. This activity is not standard use with this binary for a user or system administrator. It indicates a potentially malicious actor attempting to improve stability of their access or elevate privileges.
+"""
+from = "now-9m"
+index = ["logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Linux Restricted Shell Breakout via  apt/apt-get Changelog Escape"
+references = [
+    "https://gtfobins.github.io/gtfobins/apt/",
+    "https://gtfobins.github.io/gtfobins/apt-get/"
+]
+risk_score = 47
+rule_id = "8fed8450-847e-43bd-874c-3bbf0cd425f3"
+severity = "medium"
+tags = ["Elastic", "Host", "Linux", "Threat Detection", "Privilege Escalation", "GTFOBins"]
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = '''
+sequence by host.id, process.pid with maxspan=1m
+[process where process.name : ("apt", "apt-get") and process.args : "changelog"]
+[process where process.parent.name : ("apt", "apt-get") and process.name : "sensible-pager" and process.args : "/bin/sh"]
+'''
+
+
+[[rule.threat]]
```

Comment:
https://attack.mitre.org/techniques/T1127/ sudo and sudoedit and privesc seems to be not in the rule scope.

Reply:
Corrected